sdcm/sct_events/health.py

```diff
@@ -40,27 +42,35 @@ def __init__(self,
                  severity=Severity.ERROR) -> None:
         super().__init__(severity=severity)
 
-        self.node = str(node)
+        self.node = str(node) if node else ""
         self.error = error if error else ""
         self.message = message if message else ""
 
     @property
     def msgfmt(self) -> str:
+        message = super().msgfmt
+        if self.type:
+            message = message + ": type={0.type}"
+        if self.node:
+            message = message + " node={0.node}"
         if self.severity in (Severity.NORMAL, Severity.WARNING, ):
-            return super().msgfmt + ": type={0.type} node={0.node} message={0.message}"
+            if self.message:
+                message = message + " message={0.message}"
+            return message
         elif self.severity in (Severity.ERROR, Severity.CRITICAL, ):
-            return super().msgfmt + ": type={0.type} node={0.node} error={0.error}"
-        return super().msgfmt
-
-
-ClusterHealthValidatorEvent.add_subevent_type("NodeStatus")
-ClusterHealthValidatorEvent.add_subevent_type("NodePeersNulls")
-ClusterHealthValidatorEvent.add_subevent_type("NodeSchemaVersion")
-ClusterHealthValidatorEvent.add_subevent_type("NodesNemesis")
-ClusterHealthValidatorEvent.add_subevent_type("MonitoringStatus")
-ClusterHealthValidatorEvent.add_subevent_type("ScyllaCloudClusterServerDiagnostic")
-ClusterHealthValidatorEvent.add_subevent_type("Done", severity=Severity.NORMAL)
-ClusterHealthValidatorEvent.add_subevent_type("Info", severity=Severity.NORMAL)
+            if self.error:
+                message = message + " error={0.error}"
+            return message
+        return message
+
+
+ClusterHealthValidatorEvent.add_subevent_type("NodeStatus", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("NodePeersNulls", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("NodeSchemaVersion", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("NodesNemesis", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("MonitoringStatus", mixin=ClusterHealthValidatorSubEvents)
+ClusterHealthValidatorEvent.add_subevent_type("ScyllaCloudClusterServerDiagnostic",
```

Comment:
It's too hacky :)

All these subtypes will be informational and continuous same time.

Reply:
It's reported as informational, not continuous